hub2labhook/gitlab/client.py

```diff
@@ -173,35 +187,22 @@ def get_or_create_project(self, project_name, namespace=None,
         group_id = self.get_namespace_id(group_name)
         path = self._url("/projects")
         body = {
-            "name":
-                project_name,
-            "namespace_id":
-                group_id,
-            "issues_enabled":
-                self.config.gitlab['enable_issues'],
-            "merge_requests_enabled":
-                self.config.gitlab['enable_merge_requests'],
-            "jobs_enabled":
-                self.config.gitlab['enable_jobs'],
-            "wiki_enabled":
-                self.config.gitlab['enable_wiki'],
-            "snippets_enabled":
-                self.config.gitlab['enable_snippets'],
-            "container_registry_enabled":
-                self.config.gitlab['enable_container_registry'],
-            "shared_runners_enabled":
-                self.config.gitlab['enable_shared_runners'],
-            "public":
-                repo_public,
-            "visibility": ("public"
-                           if repo_public else self.config.gitlab['privacy']),
-            "public_jobs":
-                repo_public,
+            "name": project_name,
+            "namespace_id": group_id,
+            "builds_access_level": 'enabled',
+            "operations_access_level": 'enabled',
+            "merge_requests_access_level": 'enabled',
+            "public_builds": False,
+            'repository_access_level': 'enabled',
+            'shared_runners_enabled': False,
+            "visibility": "private"
```

Comment:
It's not needed to use the `self.config.gitlab` dict anymore?

Reply:
I'm trying to simplify a little the code. but I can make it configurable again... Also since no one is using this API except Kubespray and me (AFAIK), I tend to prefer to go easy on configuration and then add more on demand. 
WDYT?

Reply:
Sounds good.